app/components/tracks/TracksController.js

```diff
@@ -31,6 +33,57 @@ tracksModule.controller('TracksController',
                 }, function (error) {
                     console.error(error);
                 });
+
+            ApiJsonFactory.getJson('sessions')
+                .then(function (response) {
+                    tc.Sessions = response.data.sessions;
+                    $sessionStorage.sessions = tc.Sessions;
+                }, function (error) {
+                    console.error(error);
```

Comment:
My point is we should make this as a separate issue, because all the controller scripts have to be changed to make the modification because all the scripts are using console